src/lib/component/view/EventFiltersView.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from "/vendor/react";
+import gql from "/vendor/graphql-tag";
+import { ApolloError } from "/vendor/apollo-client";
+
+import { PollingDuration } from "/lib/constant";
+import { FailedError } from "/lib/error/FetchError";
+
+import {
+  parseIntParam,
+  parseStringParam,
+  parseArrayParam,
+} from "/lib/util/params";
+
+import {
+  useSearchParams,
+  SearchParamsMap,
+  useQuery,
+  useFilterParams,
+  UseQueryResult,
+  useRouter,
+} from "/lib/component/util";
+import {
+  MobileFullWidthContent,
+  Content,
+  FilterList,
+} from "/lib/component/base";
+import {
+  AppLayout,
+  EventFiltersList,
+  eventFiltersListFragments,
+  EventFiltersListToolbar,
+  NotFound,
+} from "/lib/component/partial";
+import { EventFiltersListVariables } from "/lib/component/partial/EventFiltersList/EventFiltersList";
+
+interface Variables extends EventFiltersListVariables {
+  namespace: string;
+}
+
+export const eventFiltersViewFragments = {
+  namespace: gql`
+    fragment eventFiltersView_namespace on Namespace {
+      ...EventFiltersList_namespace
+    }
+
+    ${eventFiltersListFragments.namespace}
+  `,
+};
+
+export const EventFiltersViewQuery = gql`
```

Comment:
nit: does this need to be exported?

Reply:
whoops -- yeah we don't. Will remove. Thanks for catching